docs/install/install.rst

```diff
@@ -1,73 +1,72 @@
 
 .. meta::
-  :description: Install ROCm Bandwidth Test
-  :keywords: install, rocm bandwidth test, AMD, ROCm
+  :description: ROCm Bandwidth Test is a ROCm application for reporting system information
+  :keywords: Install ROCm Bandwidth Test, Build ROCm Bandwidth Test, Install RBT, Build RBT
 
 
-Building the environment
---------------------------
+Installing ROCm Bandwidth Test
+-----------------------------------
 
-To build ROCmBandwidthTest, users must ensure that the build platform has the following conditions satisfied.
+This topic provides information required to build and install RBT.
 
+Build
+=======
 
-Build the procedure
-====================
+Here are the steps to build RBT:
 
-This section lists the steps to build ROCmBandwidthTest.
+  1. Create a build directory in the project folder - ``roc_bandwidth_test``.
 
-1. Create a build directory in the project folder - `roc_bandwidth_test`.
-
-.. code-block:: shell
+  .. code-block:: shell
 
     mkdir ./build
 
+  2. Set working directory to the new build directory.
 
-2. Set working directory to be the new build directory.
-
-.. code-block:: shell
+  .. code-block:: shell
 
     cd ./build
 
+  3. Invoke ``Cmake`` to interpret build rules and generate native build files. Provide the RBT test suite as the argument for ``Cmake``.
 
-
-3. Invoke Cmake to interpret build rules and generate native build files. The argument for cmake should be the root folder of ROCmBandwidthTest test suite.
-
-.. code-block:: shell
+  .. code-block:: shell
 
     # Assume that ROCR Runtime has its libraries & headers are located in the path :
     # libraries : _ABSOLUTE_PATH_TO_ROCR_LIBS_/lib
     # headers   : _ABSOLUTE_PATH_TO_ROCR_LIBS_/include/hsa
     # Note : Observe that both include & lib folder are under common path (_ABSOLUTE_PATH_TO_ROCR_LIBS_)
-    
+
     # Builds Debug version
     # Assumes pwd is _ABSOLUTE_PATH_TO_RBT/roc_bandwidth_test/build
 
     cmake -DCMAKE_BUILD_TYPE="Debug"   \
         -DCMAKE_MODULE_PATH="_ABSOLUTE_PATH_TO_RBT/rocm_bandwidth_test/cmake_modules"  \
         -DCMAKE_PREFIX_PATH="_ABSOLUTE_PATH_TO_ROCR_LIBS_"
```

Comment:
```suggestion
        -DCMAKE_MODULE_PATH="_ABSOLUTE_PATH_TO_RBT/rocm_bandwidth_test/cmake_modules"  \
        -DCMAKE_PREFIX_PATH="_ABSOLUTE_PATH_TO_ROCR_LIBS_"
```

Should the path be ""_ABSOLUTE_PATH_TO_RBT/roc_bandwidth_test/...." ? We have mentioned in step 1 to build in project folder roc_bandwidth_test.

Reply:
Those are different paths; `CMAKE_MODULE_PATH` is the full (absolute) path to the `cmake_modules` directory, for example: `.../rocm_bandwidth_test/cmake_modules` . 

The project folder should be `rocm_bandwidth_test` (not `roc_bandwidth_test`).

Reply:
Corrected the typo in project name to rocm_bandwidth_test.

Reply:
It looks good to me. LGTM.